### ISSUE TITLE
Transition from TEI P4 to P5 as authoring format

### ISSUE THREAD

**Issue by Conal-Tuohy** [IUBLibTech/newton_chymistry#55]
- [x] Set up P5 Xubmit repo and provide URL to @Conal-Tuohy 
- [x] Generate new P5 texts and populate the new repo with at least some files
- [x] Add a step to the app so that you can download P5 directly (@Conal-Tuohy)
- [x] Test

**Comment by mdalmau** [comment]
@Conal-Tuohy a new Xubmit repo has been created and is ready for you to perform the most current P4 to P5 transform and upload to the new repo: Chymistry of Isaac Newton, Next Gen (newtonchym). I will alert the team to freeze all P4 encoding as of now (5:30 pm EST).  Once the p5 is in place, I will work on updating the links to the manifest so we can finish testing the IIIF integration.  Thank you!

**Comment by Conal-Tuohy** [comment]
I've generated the P5 but couldn't upload them to Xubmit. Sent an email with the details. BTW I don't have access to polar, and I've only ever accessed Xubmit "properly" through its web interface.

**Comment by mdalmau** [comment]
Still troubleshooting Xubmit.  Michelle and Nick have done a fair bit of digging but aren't getting to the bottom of it.  The Xubmit directory that contains all the schemas on which it validates against upon submission to the repo is using the p5 all schema that Michelle generated, but that is still not working.  @randalldfloyd is now looking at deeper issues.  We are still at a standstill here. Sigh.

**Comment by randalldfloyd** [comment]
@mdalmau Just a question to help clarify for myself: technically speaking, if it came down to it, can we update the TEI to embed the PURL before solving the Xubmit issue so that it isn't a complete blocker?  I realize the ideal worklfow is to get them in there and create version provenance ASAP, but Xubmit could be sidestepped for just that edit, which would get the UV working nicely with images for all the stakeholders.  But yes I am currently the last cube wall that the ball can be tossed into for the underlying Xubmit problem.

**Comment by Conal-Tuohy** [comment]
@randalldfloyd as far as I can tell the Xubmit schema validation issue and the issue of the embedding IIIF manifest PURLs in the TEI P5 are entirely unrelated; I don't think there's any dependency one way or another. I'm going to add the PURL update shortly.

**Comment by mdalmau** [comment]
@Conal-Tuohy just an FYI that I don't see the PURLs to the IIIF manifest in the latest TEIP5 batch you generated.  This came up at our Newton meeting yesterday as  the team continues moving forward with editorial work while the new site gets up and going.

**Comment by Conal-Tuohy** [comment]
thanks! I will check it

**Comment by Conal-Tuohy** [comment]
I have checked the ZIP file I sent on the 9th of Feb and there are iiif-manifest references in there, e.g.

```xml
<altIdentifier>
   <idno type="iiif-manifest">https://purl.dlib.indiana.edu/iudl/newton/iiif/ALCH00001</idno>
</altIdentifier>
```

**Comment by mdalmau** [comment]
I populated the Xubmit repo with the p5 files.  I think I just need to re-index the new site with the latest P5 versions to make sure all looks good.

**Comment by mdalmau** [comment]
All files are in p5 now in the new Xubmit repo.  I need to still transform ALCH00084, but will close this issue since that's a separate issue on my to-do list!
